Cargo.toml

```diff
@@ -17,7 +17,7 @@ binary-install = "0.0.3-alpha"
 chrome-devtools-rs = { version = "0.0.0-alpha.0", features = ["color"] }
 chrono = "0.4.9"
 clap = "2.32.0"
-cloudflare = "0.6.4"
+cloudflare = { path = "../cloudflare-rs/cloudflare" }
```

Comment:
ik this is wip but one thing i like to do when developing libraries in parallel is to comment out the `path` loading for a local dependency and replace it with a `git = https://github.com/cloudflare/cloudflare-rs` and a `rev = 7897482390` when i push it up so it's clear what code it's dependent on